.travis.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  - docker
+
+sudo: false
+
+script:
+  - scripts/test.sh
+
+# This isn't used, but makes slightly more sense than ruby, the default.
+language: java
```

Comment:
If it isn't used, could we just not specify it? I feel like fewer keys is just easier.